### PR TITLE
feat(RELEASE-1171): create integration test  for release-service

### PR DESCRIPTION
**Pull request by jinqi7**

### CODE REVIEW

File: integration-tests/pipelines/konflux-e2e-tests-pipeline.yaml [hunk 1]
```diff
@@ -0,0 +1,253 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: konflux-e2e-tests-pipeline.yaml
```
Comment (flacatus) [inline integration-tests/pipelines/konflux-e2e-tests-pipeline.yaml:5]: konflux-e2e-tests-pipeline

File: integration-tests/pipelines/konflux-e2e-tests-pipeline.yaml [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+      type: string
+      description: The key secret from konflux-test-infra-secret where all AWS ROSA configurations are stored.
+    - name: replicas
+      description: 'The number of replicas for the cluster nodes.'
+      type: string
+    - name: machine-type
+      description: 'The type of machine to use for the cluster nodes.'
+      type: string
+    - name: oci-container-repo
+      default: 'quay.io/konflux-test-storage/release-service'
```
Comment (flacatus) [inline integration-tests/pipelines/konflux-e2e-tests-pipeline.yaml:39]: quay.io/konflux-test-storage/konflux-team/release-service

**Review by flacatus — CHANGES_REQUESTED** [review]

File: integration-tests/pipelines/konflux-e2e-tests-pipeline.yaml (hunk 1 above)
Comment (jinqi7) [reply integration-tests/pipelines/konflux-e2e-tests-pipeline.yaml]: Update!

File: integration-tests/pipelines/konflux-e2e-tests-pipeline.yaml (hunk 2 above)
Comment (jinqi7) [reply integration-tests/pipelines/konflux-e2e-tests-pipeline.yaml]: Updated, thanks!

File: integration-tests/pipelines/konflux-e2e-tests-pipeline.yaml [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+          value: $(params.oci-container-repo)
+        - name: oci-container-tag
+          value: $(context.pipelineRun.name)
+    - name: provision-rosa
+      when:
+        - input: "$(tasks.test-metadata.results.test-event-type)"
+          operator: in
+          values: ["pull_request"]
+      runAfter:
+        - rosa-hcp-metadata
```
[the remark this answers is not in the document]
Comment (jinqi7) [reply integration-tests/pipelines/konflux-e2e-tests-pipeline.yaml:98]: It's already been added before "provision-rosa" task.

[the remark this answers is not in the document]
Comment (flacatus) [reply integration-tests/pipelines/konflux-e2e-tests-pipeline.yaml:98]: I mean in the runAfter section

[the remark this answers is not in the document]
Comment (jinqi7) [reply integration-tests/pipelines/konflux-e2e-tests-pipeline.yaml:98]: Updated!

**Comment by kasemAlem** [comment]
it's failing on the task [quality-dashboard-upload]([quality-dashboard-upload](https://console.redhat.com/application-pipeline/ns/rhtap-release-2-tenant/pipelinerun/release-service-gn7qf-dl8sb/logs/quality-dashboard-upload)) , should we create a ticket for it, @psturc  @flacatus ?

**Comment by kasemAlem** [comment]
> it's failing on the task quality-dashboard-upload , should we create a ticket for it, @psturc @flacatus
okay, I see that Jing already fixed it according to [this](https://redhat-internal.slack.com/archives/C02FANRBZQD/p1727272933201259)

File: integration-tests/pipelines/konflux-e2e-tests-pipeline.yaml [hunk 4]
```diff
@@ -0,0 +1,254 @@
...
+          value: "$(tasks.test-metadata.results.git-revision)"
+        - name: oras-container
+          value: "$(tasks.create-oci-container.results.oci-container)"
+        - name: job-spec
+          value: "$(tasks.test-metadata.results.job-spec)"
+        - name: ocp-login-command
+          value: "$(tasks.provision-rosa.results.ocp-login-command)"
+        - name: component-image
+          value: "$(tasks.test-metadata.results.container-image)"
+        - name: container-image
```
[the remark this answers is not in the document]
Comment (jinqi7) [reply integration-tests/pipelines/konflux-e2e-tests-pipeline.yaml]: Done

**Comment by flacatus** [comment]
LGTM

**Review by flacatus — APPROVED** [review]

**Comment by jinqi7** [comment]
/hold
Currently there is some issue with "provision rosa" in e2e-tests repo, May meet the same issue here.

**Pull request merged**